be/src/storage/tablet_updates.cpp

```diff
@@ -2434,4 +2446,39 @@ Status TabletUpdates::get_column_values(std::vector<uint32_t>& column_ids, bool
     return Status::OK();
 }
 
+Status TabletUpdates::prepare_partial_update_states(Tablet* tablet, const std::vector<ColumnUniquePtr>& upserts,
+                                                    EditVersion* read_version, uint32_t* next_rowset_id,
+                                                    std::vector<std::vector<uint64_t>*>* rss_rowids) {
+    std::lock_guard wl(_lock);
```

Comment:
_lock is used in many places, it's not necessary for the whole method body

Reply:
done，https://github.com/StarRocks/starrocks/pull/2620/commits/5fdddf7ad2047c6d61b4e008ef8f0a6124676bf1

Reply:
> _lock is used in many places, it's not necessary for the whole method body

done，https://github.com/StarRocks/starrocks/pull/2620/commits/5fdddf7ad2047c6d61b4e008ef8f0a6124676bf1